src/ImageNode.cpp

```diff
@@ -1,75 +1,37 @@
 #include "ImageNode.h"
-#include "ProbeReg.h"
+#include "Context.h"
+#include "Audio.h"
 #include <QDebug>
 #include <QDir>
 #include <QFile>
 #include <QFileInfo>
 #include <QImageReader>
 #include <QtQml>
 #include "Paths.h"
-#include "main.h"
 
-
-ImageType::ImageType(NodeRegistry *r , QObject *p )
-    : NodeType(r,p) {
-}
-ImageType::~ImageType() = default;
-VideoNode *ImageType::create(QString arg) {
-    auto node = new ImageNode(this);
-    if (node) {
-        node->setInputCount(inputCount());
-        node->setImagePath(name());
-    }
-    return node;
-}
-
-ImageNode::ImageNode(NodeType *nr)
-    : VideoNode(nr)
+ImageNode::ImageNode(Context *context, QString file)
+    : VideoNode(context)
     , m_ready(false) {
+    setInputCount(1);
+    setFile(file);
 }
 
 ImageNode::ImageNode(const ImageNode &other)
     : VideoNode(other)
 //    , m_frameTextures(other.m_frameTextures)
```

Comment:
This file has a lot of commented-out code that should just be deleted

Reply:
Yeah it could use some cleanup